src/main/proto/bio/terra/common/iam/iam.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package bio.terra.common.iam;
+option java_package = "bio.terra.common.iam.proto";
+option java_multiple_files = true;
+
+message AuthenticatedUserRequestModel {
+  optional string email = 1;
+  optional string subject_id = 2;
+  optional string token = 3;
+}
+
+// DON'T DO THIS
+// This is not the correct way to change a proto schema - you would always modify the message in
+// place, e.g. add the 'additional_field'. For illustrative purposes only, we define two schemas to
+// be compiled into the same binary to show forward/backwards compatibility.
+message AuthenticatedUserRequestModelV2 {
```

Comment:
I have never try clients depends on proto file. Is `bio.terra.common.iam.proto.AuthenticatedUserRequestModel` would automattly built into published package? (I am thinking about the pain when we have to upload/download yaml file)

Reply:
The generated proto java classes are included in the jar the same as any other class.
With regards to sharing .proto files in a better way than we're doing with the upload/download yaml for open api schema, you can depend on the .proto files of a dependency directly in addition to just it's published classes. This makes it possible to share & version control .proto files as a normal part of package dependencies. So, no more pain, I think, but haven't tried for myself.
See https://github.com/google/protobuf-gradle-plugin#protos-in-dependencies

To be clear, the .proto files used only for stairway serialization shouldn't need to be shared. It's fine to use the java generated classes, or just leave it to the implementation of the shared class (`AuthenticatedUserRequest`).